Documentation/ApiOverview/ContentElements/Index.rst

```diff
@@ -28,53 +28,168 @@ Introduction
 ..  contents::
     :local:
 
-..  term clarification:
-..  1. "content elements" is used as general term, including plugins
-..  2. use "content element types" when talking about the type
+In TYPO3, content elements and plugins are both used to present and manage
+content on a website, but they serve different purposes and have distinct
+characteristics:
+
+..  card-grid::
+    :columns: 1
+    :columns-md: 2
+    :card-height: 100
+
+    ..  card:: :ref:`Content elements <content-elements>`
+
+        ..  card-image:: /Images/ManualScreenshots/Backend/content_element_example.png
+            :alt: Content element Text & Media
+
+        A content element is a standard unit for managing and displaying content,
+        such as text, images, videos, tables, and more.
```

Comment:
```suggestion
        such as text, images, videos, tables and more.
```

Reply:
This is a "oxford coma" and therefore needed